docs/gaia/installation.md

```diff
@@ -40,6 +40,37 @@ $ gaiad version
 $ gaiacli version
 ```
 
+### Install binary distribution via snap (Linux only)
+
+Gaia can be installed on various GNU/Linux distributions from the [Snapcraft.io](https://snapcraft.io/gaia) store:
+
+```bash
+$ sudo snap install gaia
```

Comment:
Is there a way to install a particular version, e.g. `gaia@0.31.0`, or not?

Reply:
Long story short: no, you can't unless you have developer access:

```bash
$ snap install --help | grep -nC3 revision
9-With no further options, the snaps are installed tracking the stable channel,
10-with strict security confinement.
11-
12:Revision choice via the --revision override requires the the user to
13-have developer access to the snap, either directly or through the
14-store's collaboration feature, and to be logged in (see 'snap help login').
15-
16:Note a later refresh will typically undo a revision override, taking the snap
17:back to the current revision of the channel it's tracking.
18-
19-Use --name to set the instance name when installing from snap file.
20-
--
30-      --devmode                       Put snap in development mode and disable security confinement
31-      --jailmode                      Put snap in enforced confinement mode
32-      --classic                       Put snap in classic mode and disable security confinement
33:      --revision=                     Install the given revision of a snap, to which you must have developer access
34-      --dangerous                     Install the given snap file even if there are no pre-acknowledged signatures for it, meaning it was not verified and could be dangerous (--devmode implies this)
35-      --unaliased                     Install the given snap without enabling its automatic aliases
36-      --name=                         Install the snap file under the given instance name
```

We should sit down and talk about snaps channels handling and promotion policies, for instance one could be as follows: 
* `edge` is the **use-at-your-own-risk** bleeding edge as it's continuosly built from the latest `develop`.
* We may want to promote RC releases to `beta` - this would make the `beta` channel turn into a `stable+1` channel
* `stable` would track the most recent stable release
* we could push stable branch hotfix releases to the `candidate` channel for testing purposes before promoting them to stable

Reply:
Hmm, that's unfortunate, installing exact versions would be more useful.

Reply:
This does support installing exact versions AFAIK right @alessio?

Reply:
Nope, there's only version available for channel

Reply:
@cwgoes I was wrong!

https://blog.ubuntu.com/2017/09/07/controlling-snap-releases-with-channels-tracks-and-branches-part-2

Reply:
Cool! Let's document that (could be a separate PR)